src/Storages/System/StorageSystemDictionaries.cpp

```diff
@@ -20,6 +20,72 @@
 
 namespace DB
 {
+namespace
+{
+    /// Used for column indexes!  Don't change the numbers!
+    enum SystemDictionariesColumn
```

Comment:
Need to fix indentation. In anonymous namespace there are no indentation.

Reply:
Fixed it, although it was done by the auto-formatter according to [this](https://github.com/ClickHouse/ClickHouse/blob/245899c0f66e4f3e5d3d833dc3cd2ffc5cbd6cb6/.clang-format#L82). BTW in the project in different places differently